sky/templates/kubernetes-ray.yml.j2

```diff
@@ -864,7 +861,7 @@ available_node_types:
                 $(prefix_cmd) mkdir -p ~/.ssh;
                 $(prefix_cmd) chown -R $(whoami) ~/.ssh;
                 $(prefix_cmd) chmod 700 ~/.ssh;
-                $(prefix_cmd) cat /etc/secret-volume/ssh-publickey* > ~/.ssh/authorized_keys;
+                echo 'skypilot:ssh_public_key_content' | $(prefix_cmd) tee ~/.ssh/authorized_keys > /dev/null;
                 $(prefix_cmd) chmod 644 ~/.ssh/authorized_keys;
                 $(prefix_cmd) service ssh restart;
```

Comment:
This will render as:

```
echo 'ssh-rsa ...'
      | $(prefix_cmd) tee ~/.ssh/authorized_keys > /dev/null;\n  $(prefix_cmd) chmod
      644 ~/.ssh/authorized_keys;\n  $(prefix_cmd) service ssh restart;
```